sdv/data_processing/data_processor.py

```diff
@@ -75,6 +76,23 @@ def _update_numerical_transformer(self, enforce_rounding, enforce_min_max_values
         )
         self._transformers_by_sdtype.update({'numerical': custom_float_formatter})
 
+    def _detect_multi_column_transformers(self):
+        """Detect if there are any multi column transformers in the metadata.
+
+        Returns:
+            dict:
+                A dictionary mapping column names to the multi column transformer.
+        """
+        result = {}
+        for relationship in self.metadata.column_relationships:
+            column_names = tuple(relationship['column_names'])
+            relationship_type = relationship['type']
+            if relationship_type == 'address':
+                transformer = rdt.transformers.address.RandomLocationGenerator
+                result[column_names] = transformer(locales=self._locales)
```

Comment:
should we make a dictionary mapping the relationship types to the default transformer to use?

Reply:
I thought of this first but I was getting import errors when defining the dict when the transformer is not available. That's why I kept it in `if`.